pkg/util/mount/mount.go

```diff
@@ -32,7 +37,19 @@ type Interface interface {
 	List() ([]MountPoint, error)
 	// IsMountPoint determines if a directory is a mountpoint.
 	IsMountPoint(file string) (bool, error)
+	SetRunner(executor ContainerExecutor)
 }
+type ContainerExecutor interface {
+	RunInContainerBySelector(selector labels.Selector, containerName string, cmd []string) ([]byte, error)
+}
+
+const (
+	MOUNTER_MOUNT = iota
```

Comment:
enums should be camel case (more go-idiomatic) and the first should be typed, e.g.:

```
const (
        MounterMount MounterType = iota
        MounterNSEnter
        ...
```